sdk/core/core-http/src/queryCollectionFormat.ts

```diff
@@ -5,9 +5,24 @@
  * The format that will be used to join an array of values together for a query parameter value.
  */
 export enum QueryCollectionFormat {
+  /**
+   * CSV: Each pair of segments joined by a single comma.
+   */
   Csv = ",",
+  /**
+   * SSV: Each pair of segments joined by a single whitespace.
+   */
   Ssv = " ",
+  /**
+   * TSV: Each pair of segments joined by a single tab.
+   */
   Tsv = "\t",
+  /**
+   * Pipes: Each pair of segments joined by a single pipe character.
+   */
   Pipes = "|",
+  /**
+   * Pairs of segments may join with multiple formats.
```

Comment:
Close! This one is tricky, you have to know how we serialize query params and the URL handling for that is non-trivial to reason about, especially in core-http. 😄 

```suggestion
   * Denotes this is an array of values that should be passed to the server in multiple key/value pairs, e.g. `?queryParam=value1&queryParam=value2`
```

Reply:
oooh! thank you!